internal/cli/internal/command/machine/list.go

```diff
@@ -55,54 +57,44 @@ func runMachineList(ctx context.Context) (err error) {
 	var (
 		appName = app.NameFromContext(ctx)
 		client  = client.FromContext(ctx).API()
-		stream  = iostreams.FromContext(ctx)
-		cfg     = config.FromContext(ctx)
+		io      = iostreams.FromContext(ctx)
 	)
 
-	state := flag.GetString(ctx, "state")
```

Comment:
Do we need to keep these flags if we’re not using them? For `state`, we could filter in the CLI? and get rid of `all`. Also looks like `quiet` outputs only the IDs and my guess is that’s useful for scripting.